cmd/gitlab-ci-pipelines-exporter/main.go

```diff
@@ -92,15 +106,58 @@ func main() {
 			return err
 		}
 
-		logger.Info(fmt.Sprintf("start listening on %s", ctx.String("web.listen-address")))
-		return http.ListenAndServe(
-			ctx.String("web.listen-address"),
-			webhook.NewWebhook(
-				ctx.String("web.telemetry-path"),
-				ctx.String("gitlab.webhook-secret-token"),
-				webhook.WithZapLogger(logger),
-			),
+		// Graceful shutdowns
+		onShutdown := make(chan os.Signal, 1)
+		signal.Notify(onShutdown, syscall.SIGINT, syscall.SIGTERM, syscall.SIGABRT)
```

Comment:
Why handling `SIGABRT` ?
```suggestion
		signal.Notify(onShutdown, syscall.SIGINT, syscall.SIGTERM)
```

Reply:
I missed this one. On Stack Overflow they say it's sent when an internal issue occurs. For instance the glibc will trigger this signal if a memory allocation fails or if `free()` is called with an invalid pointer.

I will remove this since we pretty much don't care